docs/rules/object-curly-spacing.md

```diff
@@ -145,5 +145,4 @@ You can turn this rule off if you are not concerned with the consistency of spac
 
 * [comma-spacing](comma-spacing.md)
 * [space-in-parens](space-in-parens.md)
-* [space-in-brackets](space-in-brackets.md) (deprecated)
-
+* [space-in-brackets](space-in-brackets.md)
```

Comment:
I wasn't sure. This is the only place where it's mentioned that it is and I don't see any rule that supersedes it.